blocks/rich-text/index.js

```diff
@@ -412,14 +395,25 @@ export class RichText extends Component {
 	 *
 	 * @param {boolean} checkIfDirty Check whether the editor is dirty before calling onChange.
 	 */
-	onChange( checkIfDirty = true ) {
-		if ( checkIfDirty && ! this.editor.isDirty() ) {
```

Comment:
I imagine this should no longer be necessary:

https://github.com/WordPress/gutenberg/blob/9cd9c180290f4b9281ee795f391b4a04586309fe/blocks/rich-text/index.js#L803

Reply:
You may be right. Don't know if it's trying to communicate something else to TinyMCE after handling formatting ourselves. I don't immediately see anything broken by removing it though.